VALIDATIONS.md

```diff
@@ -0,0 +1,36 @@
+# CI Operator Validation Reference
```

Comment:
I was told to always call the thing "ci-operator", not "CI Operator" or something similar